getPropertyProfiles.m

```diff
@@ -98,15 +98,18 @@
 options.MAP_index = 1;
 
 % Warning if objective function gradient is not available
-if ~strcmp(options.profileReoptimizationOptions.GradObj, 'on')
+if isempty(options.profileOptimizationOptions)
+    options.profileOptimizationOptions = options.localOptimizerOptions;
+end
+if ~strcmp(options.profileOptimizationOptions.GradObj, 'on')
     warning('For efficient and reliable optimization, getPropertyProfiles.m requires gradient information.')
 end
 
 %% Initialization and figure generation
 fh = [];

```

Comment:
this function (+ variations) seems to appear in multiple functions. Can't we create a seperate function that returns options+fh?

Reply:
We can and probably should. I opened #154 for this.